pkg/sources/github/github.go

```diff
@@ -255,6 +256,13 @@ func (s *Source) Init(aCtx context.Context, name string, jobID sources.JobID, so
 	s.includeIssueComments = s.conn.IncludeIssueComments
 	s.includePRComments = s.conn.IncludePullRequestComments
 	s.includeGistComments = s.conn.IncludeGistComments
+	s.commentsTimeframeDays = s.conn.CommentsTimeframeDays
+	// TODO: Cleanup
+	aCtx.Logger().Info("Set commentsTimeframeDays",
+		"commentsTimeframeDays", s.commentsTimeframeDays,
+		"conn.CommentsTimeframeDays", s.conn.CommentsTimeframeDays,
+		"conn.GetCommentsTimeframeDays", s.conn.GetCommentsTimeframeDays(),
+	)
```

Comment:
This is showing `commentsTimeframeDays` is `0` even when running `go run main.go --debug github --org=jonzeolla --fail --no-verification --issue-comments --pr-comments --comments-timeframe=1`

Any idea why?

Reply:
I'm still not sure why this isn't working, but the rest of my code could use a bit of a tuneup as well; working on that shortly.

Reply:
The issue was I needed to run `make protos` to generate `sources.pb.go`.